lib/student.rb

```diff
@@ -1,9 +1,8 @@
 require_relative 'person'
 
 class Student < Person
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+  def initialize(name, age, parent_permission: true)
     super(name, age, parent_permission)
-    @classroom = classroom
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, I'm getting an error trying to create a student:

![image](https://github.com/hajnaloltyan/school-library/assets/77069011/15808a9a-f5a3-4f75-867e-00cf1e723395)

This is because you're passing the `parent_permission` parameter wrong inside your `#create_student` method and here, when you pass `parent_permission` to the parent class, `Person`.

If you look closely, you defined `parent_permission` as a hash, so you have to pass it like a `key: value` pair:

```rb
# app.rb ln 77
person = Student.new(age, name, parent_permission: parent_permission)



# student.rb ln 5
super(name, age, parent_permission:parent_permission)
```